lib/popcorn/pop_middlewares.nit

```diff
@@ -42,23 +42,68 @@ end
 class ConsoleLog
 	super Handler
 
+	# Logger level
+	#
+	# * `0`: silent
```

Comment:
Oups! I was confusing the “logger”’s level with the message’s level. Sorry. My mistake.